java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogSnapshotStack.java

```diff
@@ -204,7 +209,8 @@ private void createSnapshotDeletionLambda(InstanceProperties instanceProperties,
         queue.grantSendMessages(snapshotDeletionTrigger);
 
         snapshotDeletionLambda.addEventSource(SqsEventSource.Builder.create(queue)
-                .batchSize(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_DELETION_BATCH_SIZE)).build());
+                .batchSize(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_DELETION_BATCH_SIZE))
+                .maxConcurrency(instanceProperties.getInt(TRANSACTION_LOG_SNAPSHOT_DELETION_LAMBDA_CONCURRENCY_MAXIMUM)).build());
```

Comment:
The snapshot deletion lambda seems to be missing the corresponding reserved concurrency setting.

Reply:
Addressed